### ISSUE TITLE
Rebell Support Squad not getting their Static Weapons.

### ISSUE THREAD

**Issue by Lazejun** [official-antistasi-community/A3-Antistasi#1551]
**Describe the bug**
When recruiting any Type of **groupsSDKSquadSupp**, so MG Squad and Mortar Squad, they spawn in, get their Static Backpacks, but then get stripped from them, you also Pay Extra for the Static weapons that they get removed.

**To Reproduce**
Buy an MG/Mortar Squad via Commander Recruit Menu.

**Expected behaviour**
The 2 Crewman per Teams should keep their Static Weapon backpacks.

Modset:
Any Modset.

**Comment by jaj22** [comment]
Yeah, ordering problem in addFIASquadHC, dates back to the FIAinit rework in 2.2. Mortar/MG backpack adding needs to be moved after the FIAinit calls.

I don't think the backpacks actually affect the static weapon placement but IIRC they do affect the dismiss/save value.

**Comment by AlexTriada** [comment]
this is fake bug, delete this bug

**Comment by AlexTriada** [comment]
why they should keep their Static Weapon backpack? if they deploy MG/Mortar???

**Comment by Lazejun** [comment]
Don't mix Team and Squad, this is about the 8 Man SQUADS that should have a Static Weapon, but they don't.

And people pay extra for the static weapons which they don't get.

And people buy them and complain cause you can't add them to a Garison

**Comment by AlexTriada** [comment]
ok, sorry, deleting wrong comment

**Comment by AlexTriada** [comment]
I didn't know about squads with static at all, my lack of knowledge about Antistasi

**Comment by RihardsVla** [comment]
i see this problem too (current version of antistasi with full RHS and RHS dependencies, nothing else). I would suggest 1. simply remove those squads as not purchasable OR 2. if people who is playing with differnet mods who observe that those squads actualy works, then somebody should write on antistsi guide what mods are needed for AI to work.
